silk/sql.py

```diff
@@ -53,7 +54,11 @@ def _explain_query(connection, q, params):
 
         # currently we cannot use explain() method
         # for queries other than `select`
-        prefixed_query = f"{prefix} {q}"
+        if re.match(f"^{prefix} .*", q):
```

Comment:
What do you think about using just `str.startswith()` instead?
```suggestion
        if q.startswith(prefix):
```

Reply:
Yeah, that will work!